File: 20-Lambdas-and-Built-In-Functions/63-Filter!.py
def remove_negatives(nums):
    return list(filter(lambda n: n >= 0, nums))

# === tests ===
print(remove_negatives([-1, 3, 4, -99]))
print(remove_negatives([-7, 0, 1, 2, 3, 4, 5]))

File: 20-Lambdas-and-Built-In-Functions/63-Filter!.js
function removeNegatives(nums) {
    return nums.filter(n => n >= 0)
}

// === tests ===
console.log(removeNegatives([-1, 3, 4, -99]))
console.log(removeNegatives([-7, 0, 1, 2, 3, 4, 5]))